src/screens/AddEdit/AddEdit.tsx

```diff
@@ -1,31 +1,251 @@
+        numbers.map((row, index) => {
+            return(
+                mapHelper(numbers[index].number, numbers[index].category, index, icons.phone, formLabels.number, numbers.length)
+            )
+        })
+    );
+
+    const mapEmails = (emails: {category: string, email: string, }[],) => (
+        emails.map((row, index) => {
+            return(
+                mapHelper(emails[index].email, emails[index].category, index, icons.email, formLabels.email, emails.length)
+            )
+        })
+    );
+
+    const mapHelper = (phoneOrEmail: string, category: string, index: number, icon: string, label: string, listLength: number) => (
+        <View key={index}>
+            <View style={styles.row}>
+                {showIconOrEmptySpace(index === 0, icon)}
+                <TextInput
+                    label={label}
+                    value={phoneOrEmail}
+                    style={styles.inputText}
+                    onChangeText={text => onChangeTextInput(label, text, index)}
+                />
+                <View style={styles.iconContainer}>
+                    {listLength > 1 && (
+                        <IconButton
+                            icon='trash-can-outline'
+                            size={30}
+                            onPress={() => onDeleteTextInput(label, index)}
+                        />
+                    )}
+                </View>
+            </View>
+            <View style={styles.row}>
+                {showIconOrEmptySpace(false, 'none')}
+                <View style={{width: '45%'}}>
+                    <RNPickerSelect
+                        value={category}
+                        onValueChange={(value) => onChangeDropdown(label, value, index)}
+                        items={contactLabels}
+                    />
+                </View>
+            </View>
+        </View>
+    );
+
+    const plusButton = (label: string) => (
+        <View style={{...styles.icon, alignSelf: "center"}}>
+            <IconButton
+                icon='plus'
+                onPress={() => addInputField(label)}
+                size={30}
+            />
+        </View>
+    );
+
+    const showSnackbar = () => (
+        snackbar.isActionVisible ? (
+            <Snackbar
+                visible={snackbar.isVisible}
+                style={styles.snackbar}
+                onDismiss={onDismissSnackbar}
+                action={{
+                    label: 'Undo',
+                    onPress: () => onUndoPressed(snackbar.label)
+                }}
+            >
+                {snackbar.message}
+            </Snackbar>
+        ) : (
+            <Snackbar
+                visible={snackbar.isVisible}
+                style={styles.snackbar}
+                onDismiss={onDismissSnackbar}
+            >
+                {snackbar.message}
+            </Snackbar>
+            )
+    );
+
+    const groupsButton = () => (
+        <Button title={"Grupy"}
```

Comment:
Język

Reply:
Grupy są jeszcze nieruszone, nie w tym pull'u

Reply:
Aaa dobra to luz